pallets/subtensor/src/staking/stake_utils.rs

```diff
@@ -1143,10 +1223,12 @@ impl<T: Config> Pallet<T> {
                 TransferToggle::<T>::get(origin_netuid),
                 Error::<T>::TransferDisallowed
             );
-            ensure!(
-                TransferToggle::<T>::get(destination_netuid),
-                Error::<T>::TransferDisallowed
-            );
+            if origin_netuid != destination_netuid {
```

Comment:
Saves one read

We need to make sure we check only for coldkey dest != coldkey origin